adapter/config/server/apiserver.go

```diff
@@ -12,7 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package apiserver
```

Comment:
Can you rename this folder to config/API so that we can have server.go and client.go underneath it?